ui/partials/perf/comparerror.html

```diff
@@ -0,0 +1,6 @@
+<div class="alert alert-danger" role="alert">
+    <p ng-repeat="error in errors">
+        <span class="fa fa-warning"></span> <span class="revision-holder">{{error}}</span>
```

Comment:
So, is that mean we can merge it for now? And we can do the separate work on the other bug.